site/source/api_items.py

```diff
@@ -116,7 +116,6 @@ def get_mapped_items():
 	mapped_wiki_inline_code['emscripten_set_beforeunload_callback']=':c:func:`emscripten_set_beforeunload_callback`'
 	mapped_wiki_inline_code['emscripten_random()']=':c:func:`emscripten_random`'
 	mapped_wiki_inline_code['EMSCRIPTEN_RESULT']=':c:macro:`EMSCRIPTEN_RESULT`'
-	mapped_wiki_inline_code['emscripten_get_canvas_size']=':c:func:`emscripten_get_canvas_size`'
```

Comment:
this file is autogenerated I think, you don't need to touch it. i don't think it hurts though.

Reply:
Oh hmm, it is in `site/source/`, but either way is ok.